pkgs/by-name/xe/xenia-canary/package.nix

```
@@ -0,0 +1,78 @@
+{
+  lib,
+  python3,
+  gtk3,
+  lz4,
+  SDL2,
+  pkg-config,
+  vulkan-loader,
+  ninja,
+  cmake,
+  libuuid,
+  llvmPackages_18,
+  autoPatchelfHook,
+  unstableGitUpdater,
+  fetchFromGitHub,
+}:
+llvmPackages_18.stdenv.mkDerivation {
+  pname = "xenia-canary";
+  version = "0-unstable-2025-06-07";
+
+  src = fetchFromGitHub {
+    owner = "xenia-canary";
+    repo = "xenia-canary";
+    fetchSubmodules = true;
+    rev = "422517c673bba086c2b857946ae5a37ee35b8e50";
+    hash = "sha256-88GHKXURfN8vaVNN7wKn562b6FvsIm/sTcUgtuhvVxM=";
+  };
+
+  dontConfigure = true;
+
+  nativeBuildInputs = [
+    python3
+    pkg-config
+    ninja
+    cmake
+    autoPatchelfHook
+    libuuid
```

```suggestion
    libuuid
    wrapGAppsHook3
    copyDesktopItems
```

`wrapGAppsHook3` is to prevent us from missing something, such as GTK built-in icons and icons in file picker. If you don't add it, someone may request you to do so later